fiftyone/constants.py

```diff
@@ -106,11 +106,3 @@
 
 # Server setup
 SERVER_DIR = os.path.join(FIFTYONE_DIR, "server")
-
```

Comment:
@benjaminpkane just confirming that this is safe to remove: it doesn't appear to be used

Reply:
Yes, no used